public/index.html

```diff
@@ -20,6 +20,28 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>React App</title>
+    <!--
+      Provide a default styling to show till the react app finishes loading. 
+    -->
+    <style type="text/css">
```

Comment:
is it possible for this style block to come from a css file?

Reply:
Nope. I've added some comments, but essentially we'd like this loading animation to be displayed as soon as possible, before even the webpack bootstrap code has been fetched and starts pulling in our js/css files. So, this has to be self contained inside the html.